### ISSUE TITLE
Handle Single-Period Events

### ISSUE THREAD

**Issue by andyl** [listcall/list_call#55]
<a href="https://github.com/andyl"><img src="https://avatars0.githubusercontent.com/u/90030?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [andyl](https://github.com/andyl)**
_Friday Feb 14, 2014 at 06:29 GMT_
_Originally opened as https://github.com/andyl/smsoesb/issues/118_

----

Disable period creation/deletion for single-period event types

**Comment by andyl** [comment]
<a href="https://github.com/marksmcsar"><img src="https://avatars2.githubusercontent.com/u/6392088?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [marksmcsar](https://github.com/marksmcsar)**
_Friday Feb 14, 2014 at 22:17 GMT_

----

Will this impact the multi-team functionality?

**Comment by andyl** [comment]
<a href="https://github.com/andyl"><img src="https://avatars0.githubusercontent.com/u/90030?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [andyl](https://github.com/andyl)**
_Saturday Feb 15, 2014 at 15:35 GMT_

----

Good question.  My initial idea for parent/child events was to tie the meta-data together.  With this scheme, when you updated the event location or time, all of the linked events (parent and child) would be auto-updated.

But as I got further into it, the rules governing meta-data update and propagation grew complex.  For example, different teams can have different event types - how to associate one with another?

So now, the meta-data for parent/child events is not linked - each event can be updated independently with no parent/child side effects.  When you create a new parent/child event, the event type is set to "TBD", and can be updated by the partner admin.

Multi-team events are complicated and we will make changes to the UI as we gain hands-on experience.

TL;DR - This proposed change is compatible with multi-team events.
